db/make_db.sh

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
```

Comment:
sorry if I missed it, but where do you initialize the database if this file is removed?